oslo_vmware/rw_handles.py

```diff
@@ -658,6 +738,7 @@ def close(self):
         :raises: VimException, VimFaultException, VimAttributeException,
                  VimSessionOverLoadException, VimConnectionException
         """
+        self._conn.close()
```

Comment:
This is already achieved down in the `finally`. If you follow the path it leads to `FileHandle.close()` which closes the same socket (`self.file_handle`).